maro/rl/rollout/env_sampler.py

```diff
@@ -45,23 +45,27 @@ def set_policy_state(self, policy_state_dict: Dict[str, object]) -> None:
             if isinstance(policy, RLPolicy):
                 policy.set_state(policy_state)
 
-    def choose_actions(self, state_by_agent: Dict[Any, np.ndarray]) -> Dict[Any, np.ndarray]:
+    def choose_actions(
+        self, state_by_agent: Dict[Any, Union[np.ndarray, List[object]]]
+    ) -> Dict[Any, Union[np.ndarray, List[object]]]:
         """Choose action according to the given (observable) states of all agents.
 
         Args:
-            state_by_agent (Dict[Any, np.ndarray]): Dictionary containing each agent's state vector.
-                The keys are agent names.
+            state_by_agent (Dict[Any, Union[np.ndarray, List[object]]]): Dictionary containing each agent's states.
+                If the policy is a `RLPolicy`, its state is a Numpy array. Otherwise, its state is a list of objects.
 
         Returns:
-            actions (Dict[Any, np.ndarray]): Dict that contains the action for all agents.
+            actions (Dict[Any, Union[np.ndarray, List[object]]]): Dict that contains the action for all agents.
```

Comment:
If the policy is a `RLPolicy`, its return value is a Numpy array. Otherwise, its return value is a list of objects.